integration/vscode/ada/package.json

```diff
@@ -417,20 +417,37 @@
         "commands": [
             {
                 "command": "ada.otherFile",
-                "title": "Go to other Ada file"
+                "title": "Ada: Go to other Ada file"
```

Comment:
let's call this one "Ada: Go to other file" ?

Reply:
Ah right, good catch